package.json

```diff
@@ -39,7 +39,7 @@
   "scripts": {
     "test": "jest ./test/test.js --coverage",
     "test:ci": "npm run lint && npm run test -- --ci --coverageReporters=json && npm run test:types",
-    "test:integration": "jest ./test/integration.test.js",
+    "test:integration": "jest ./test/*.integration.test.js",
```

Comment:
let's move them to dedicated folders so we don't have to do this trick. let's have a unit and integration folders